Resources/Prototypes/_Moffstation/Entities/Objects/Specific/Robotics/borg_modules.yml

```
@@ -30,6 +30,31 @@
   - type: BorgModuleIcon
     icon: { sprite: _Moffstation/Interface/Actions/actions_borg.rsi, state: bureau-module }
 
+# Science Modules
+- type: entity
+  id: BorgModuleAdvancedAnomaly
+  parent: [ BaseBorgModuleScience, BaseProviderBorgModule ]
+  name: advanced anomaly cyborg module
+  components:
+  - type: Sprite
+    layers:
+    - state: science
+    - sprite: _Moffstation/Objects/Specific/Robotics/borgmodule.rsi
+      state: icon-anomalies-adv
+  - type: ItemBorgModule
+    hands:
+    - item: AnomalyScanner
+    - item: WeaponPistolCHIMPBorg
+    - item: WeaponGauntletGorilla
+    - hand:
+        emptyRepresentative: AnomalyCoreBluespaceInert
+        emptyLabel: borg-slot-anomaly-core
+        whitelist:
+          tags:
+          - AnomalyCore
```

```suggestion
        whitelist:
          components:
          - AnomalyCore
```
Analogous component exists.

---

MY ENEMY, NOOOOOOOOOOOOOOOOO -

alright, let me fix this.